core/src/main/java/io/grpc/internal/RetriableStream.java

```diff
@@ -949,12 +949,8 @@ public void run() {
             if (commit) {
```

Comment:
remove early commit on hedging, too?

Reply:
It looks to me like that is correct, so have also removed the early commit on hedging.